scripts/xapi-domains.service

```diff
@@ -1,9 +1,9 @@
 [Unit]
 Description=Start/stop domains on dom0 start/shutdown
 Requires=proc-xen.mount
-Wants=xapi-init-complete.target
+Wants=xapi-init-complete.target stunnel@xapi.service
 # xapi.service is needed for shutdown ordering
-After=remote-fs.target xapi-init-complete.target xapi.service
+After=remote-fs.target xapi-init-complete.target xapi.service stunnel@xapi.service
```

Comment:
I thought that After is redundant here, but according to the docs apparently not. Wants does not introduce an ordering dependency ... so this is correct.